RCPTT_Tests/BlocksTests/BlocksAreFormattedCorrectly.test

```diff
@@ -0,0 +1,54 @@
+--- RCPTT testcase ---
+Format-Version: 1.0
+Element-Name: BlocksAreFormattedCorrectly
+Element-Type: testcase
+Element-Version: 3.0
+External-Reference: 
+Id: _Vcj1YFEBEeeEZqjlkcOTbQ
+Runtime-Version: 2.1.0.201606221726
+Save-Time: 6/14/17 3:54 PM
+Testcase-Type: ecl
+
+------=_.content-0a7243a0-75d3-3d5f-9791-539de0e5b7ac
+Content-Type: text/ecl
+Entry-Name: .content
+
+proc configure_simple_ioc [val name] {
+	let [val window [edit_current_config]] {
+ 		add_ioc -ioc_name "SIMPLE" -window $window
```

Comment:
Perhaps use add_ioc_no_macros, and add an autostart parameter. Then the code is much simpler and you don't need to pass `window` around as an argument.

```
proc configure_simple_ioc [val name] {
	with [edit_current_config] {
		add_ioc_no_macros "SIMPLE" -auto_start true
 		add_blocks_to_simple_ioc
 		save_config
 	}
 }
 
proc add_blocks_to_simple_ioc {
 	add_block "LONG" "SIMPLE:LONG" 
 	add_block "INT" "SIMPLE:VALUE1"
 	add_block "P3" "SIMPLE:VALUE:P3"
 	add_block "P5" "SIMPLE:VALUE:P5"
 	add_block "STRING" "SIMPLE:HELLO"
}
```

```
// Add an IOC without macros
proc add_ioc_no_macros [val ioc_name] [val window -input true] [val auto_start -value false]{
	with $window {
		add_ioc $ioc_name
	}
	try -times 100 -delay 200 -command {
		with [get_ioc_dialog] {
			if [$auto_start | equals true] {
				get-button "Auto-Start" | check
			}
			get-button OK | check_enabled_and_click
		}
	}
}
```